dali/python/nvidia/dali/plugin/pytorch.py

```diff
@@ -57,10 +57,12 @@ def feed_ndarray(dali_tensor, arr, cuda_stream = None):
     assert dali_tensor.shape() == list(arr.size()), \
             ("Shapes do not match: DALI tensor has size {0}"
             ", but PyTorch Tensor has size {1}".format(dali_tensor.shape(), list(arr.size())))
-    #turn raw int to a c void pointer
+    cuda_stream = types._raw_cuda_stream(cuda_stream)
+
+    # turn raw int to a c void pointer
     c_type_pointer = ctypes.c_void_p(arr.data_ptr())
     if isinstance(dali_tensor, (TensorGPU, TensorListGPU)):
-        dali_tensor.copy_to_external(c_type_pointer, cuda_stream)
+        dali_tensor.copy_to_external(c_type_pointer, ctypes.c_void_p(cuda_stream))
```

Comment:
Now pass nullptr instead of None. It break the logic in:
```
    .def("copy_to_external",
        [](Tensor<GPUBackend> &t, py::object p, py::object cuda_stream, bool non_blocking) {
          void *ptr = ctypes_void_ptr(p);
          cudaStream_t stream = cuda_stream.is_none()
                ? UserStream::Get()->GetStream(t)
                : static_cast<cudaStream_t>(ctypes_void_ptr(cuda_stream));
```
I would still pass None when no stream is provided.